src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -107,6 +106,34 @@ Self-service VPC peering setup is not supported for CockroachDB {{ site.data.pro
 
 To establish an AWS PrivateLink connection, refer to [Managing AWS PrivateLink for a cluster]({% link cockroachcloud/aws-privatelink.md %}). After the connection is established, you can use it to [connect to your cluster](#connect-to-your-cluster).
 
+#### Azure Private Link
+
+{{site.data.alerts.callout_success}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+1. Navigate to your cluster's **Networking > Private endpoint** tab.
+1. Click **Add a private endpoint**. Copy the value provided for **Alias**. Do not close this browser window.
+1. In a new browser window, log in to Azure Console and create a new private endpoint for your cluster.
+    - Set the connection method to “by resource ID or alias”.
+    - Set the resource ID to the **Alias** you previously copied. For details, refer to [Create a private endpoint](https://learn.microsoft.com//azure/private-link/create-private-endpoint-portal?tabs=dynamic-ip) in the Azure documentation.
+
+    After the private endpoint is created, copy its Resource ID. Do not close this browser window.
+1. Return to the CockroachDB {{ site.data.products.cloud }} Console browser tab and click **Next**.
+1. Paste the resource ID for the Azure private endpoint, then click **Validate**. If validation fails, verify the resource ID and try again. Otherwise, click **Next** to configure private DNS. Make a note of the Internal DNS Name and Record Name. Do not close this browser window.
```

Comment:
> Make a note of the Internal DNS Name and Record Name.

I think this should mention just Internal DNS Name. We're getting rid of Record Name.